recipes/Getting_Started/Getting_Started_with_Ollama.ipynb

```diff
@@ -81,7 +81,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!ollama pull granite-code:3b"
+    "!ollama pull granite-code:3b\n",
+    "!ollama pull granite-code:8b\n",
+    "!ollama pull granite-code:20b"
```

Comment:
Agreed, the user might as well download these three. Should we also include 34B?